src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -366,6 +372,29 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             };
         }
 
+        internal JObject GetHostJson()
```

Comment:
There's already logic in ReadDurableTaskConfig that is reading host.json. Let's not duplicate this code, instead refactoring to share the existing logic.

Reply:
What do you think, @cgillum ? Is it unblock the Durable for Kubernetes Environment?

Reply:
Since this PR is more of an "emergency" PR and we're opening a second one for the "proper" fix, I suggest we defer refactoring for the later PR (let's add a note so that we don't forget).

Reply:
I added some TODO's try not to forget.